### PR TITLE
Update timestamp identification

### PR DESCRIPTION
**Pull request by codingWithJimmy**
- Added extraction of `serverTimestamp` field
- Compared `server_timestamp` with `sent_at` to determine which one is closer to the actual send-time of the message

### CODE REVIEW

File: sigexport/data.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def fetch_data(
         res = json.loads(result[0])
         cid = result[1]
         if cid and cid in convos:
-            if res.get("type") in ["keychange", "profile-change"]:
+            if res.get("type") in ["keychange", "profile-change"]:
```
Comment (carderne) [inline sigexport/data.py:61]: Delete the space
```suggestion
            if res.get("type") in ["keychange", "profile-change"]:
```

**Review by carderne — CHANGES_REQUESTED** [review]
Some changes needed.

**Review by codingWithJimmy — COMMENTED** [review]
Thanks for that. I am not as decent at Python as I'd like to be.

File: sigexport/models.py [hunk 2]
```diff
@@ -33,7 +34,9 @@ class RawMessage:
     quote: dict[str, Any] | None
 
     def get_ts(self: RawMessage) -> int:
-        if self.sent_at:
+        if self.sent_at>self.server_timestamp:
```
[the remark this answers is not in the document]
Comment (codingWithJimmy) [reply sigexport/models.py]: Perhaps it's as simple as this then

**Review by carderne — APPROVED** [review]

**Pull request merged**